### DISCUSSION TITLE
Find Query - test if column is null

### DISCUSSION THREAD

**Discussion by elvince** [feathersjs/feathers#3484] [Q&A]
### Steps to reproduce

Create a feather app with knex.
Create a shema with a nullable/optional column
 * optId: Type.Optional(Type.Number()),

Query to filter all row where the optional column is null

### Expected behavior

Get the resutls

### Actual behavior

Can't it to work at all.
$ne: do not accept null value
If I set it to undefined, I have a binding error in knex.

Could you please help to create this query?
I read the docs and I can't get any explanation on how to write this kind of query.

Thanks,

**Comment by daffl** [comment]
```ts
optId: Type.Union([Type.Number(), Type.Null()])
```

Will make the type `number | null`.

**Reply by elvince** [reply]
Thanks for the update.
Why Type.optional is not working? Isn't it the same as your proposal?

Thanks,

**Reply by daffl** [reply]
`Type.Optional` is `undefined`, not `null`.
